packages/common/client-utils/src/base64EncodingBrowser.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-// Note: The exports map in package.json will correct this import to "./bufferNode.js" in node environments
+// Note: See 'Isomorphic Code' section in the package README.md
```

Comment:
We should also note that this file is duplicated exactly except for this import line in base64EncodingNode.ts. And have the contra statement in base64EncodingNode.ts. We want both changed at the same time.